rules.go

```diff
@@ -300,6 +302,11 @@ func handleRulesQuery(input string) string {
 		foundRuleNum := ruleRegexp.FindAllStringSubmatch(input, -1)[0][1]
 
 		log.Debug("In handleRulesQuery (Volo)", "Rules matched on", foundRuleNum)
+		if (stringSliceContains(tooLongRules, foundRuleNum)) {
+			tooLongRule := []string{"<b>", foundRuleNum, ".</b> The subtype list is too long. Please see ", voloSpecificRuleEndpointURL + foundRuleNum}
+			return strings.Join(tooLongRule, "")
```

Comment:
This is nicer as `return fmt.Sprintf("<b>%s.</b>...", foundRuleNum, ...)`

Reply:
Fix't